.github/workflows/build-opencv-base-container.yml

```diff
@@ -10,24 +10,68 @@ on:
     - build/intermediate-containers.mk
     - Makefile
   pull_request:
+    types: [assigned, opened, synchronize, reopened, labeled, unlabeled]

```

Comment:
if a user changes their PR (i assume type=edited), will this action be triggered?

Reply:
By "a user changes their PR" I assume I meant like changing title of the PR. As of now, changing the title of the PR won't trigger checking the label and commenting about missing label. I think this behavior seem reasonable. Thought?

Reply:
I found more details about edited action here https://frontside.com/blog/2020-05-26-github-actions-pull_request/

edited = title, body, or the base branch of the PR is modified